tests/cli/commands/test_jobs_command.py

```diff
@@ -109,7 +109,7 @@ def test_should_ignore_not_running_jobs(self):
             for _ in range(3):
                 scheduler_job = Job()
                 job_runner = SchedulerJobRunner(job=scheduler_job)
-                scheduler_job.state = State.SHUTDOWN
+                scheduler_job.state = State.FAILED
```

Comment:
Let’s change these `State` references to strong typed enum (e.g. `JobState` for this instance) while we’re at it.